stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -175,12 +181,12 @@ def register_post_search(self):
             response_model=(ItemCollection if not fields_ext else None)
             if self.settings.enable_response_models
             else None,
-            response_class=self.response_class,
+            response_class=GeoJSONResponse,
```

Comment:
This will cause the GeoJSONResponse to not use ORJSONResponse if the `self.response_class` is ORJSONResponse, leading to slowdowns in what's the largest API responses.

Is there a way to just set the `media_type` at response class creation time instead of setting the media type through this mechanism?

Reply:
To resolve this, I've used `importlib` to conditionally use `ORJSONResponse` when `orjson` is available and `JSONResponse` otherwise.